lmdb/lmdb.h

```diff
@@ -210,7 +239,7 @@ typedef int mdb_filehandle_t;
 	MDB_VERINT(MDB_VERSION_MAJOR,MDB_VERSION_MINOR,MDB_VERSION_PATCH)
 
 /** The release date of this library version */
-#define MDB_VERSION_DATE	"December 28, 2016"
+#define MDB_VERSION_DATE	"December 19, 2015"
```

Comment:
This is one line that always confused me about these "releases" that are on master.